### PR TITLE
Auto docker build using GitHub action

### PR DESCRIPTION
**Pull request by bam241**
this should allow to build the 6 docker image we use in CI.

It is fairly repetitive but I want to leverage docker cache build, this is why I did not build this as a matrix.

I am very open to discuss this point.

### CODE REVIEW

**Comment by gonuke** [comment]
Do we need to enable actions - this doesn't seem to be running actions

**Comment by bam241** [comment]
it is running on push, not on pull_request that's why.

but it is actually running on my fork: https://github.com/bam241/pyne/actions/runs/1037940249

**Comment by gonuke** [comment]
This doesn't seem to be making the correct combinations... is it still being tested?

**Comment by bam241** [comment]
is on my fork

https://github.com/bam241/pyne/runs/3089736086?check_suite_focus=true

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,37 +5,37 @@ version: 2.1
 executors:
     py3:
         docker:
-            - image: pyne/ubuntu_18.04_py3_pyne-deps:latest
+            - image: ghcr.io/pyne/ubuntu_18.04_py3_pyne-deps:latest
               auth:
```
Comment (gonuke) [inline .circleci/config.yml:9]: Do we still need the authorization once we move to GHCR?

**Review by gonuke — CHANGES_REQUESTED** [review]
I see the images now... I keep getting confused by the different PRs and image locations.

File: .circleci/config.yml (hunk 1 above)
Comment (bam241) [reply .circleci/config.yml:9]: no we don't

**Comment by bam241** [comment]
> Do we need to enable actions - this doesn't seem to be running actions

but you might be right, I don't understand why #1393 GitHub action is not running....

**Comment by bam241** [comment]
maybe we should hold off here before we don't know for sure....

**Comment by gonuke** [comment]
Thanks @bam241 - I'll merge this now b/c the last commit won't trigger a new image build anyway, by design

**Comment by gonuke** [comment]
> maybe we should hold off here before we don't know for sure....
> 
> 

Too late 😀 - I think i was ok to merge this

**Comment by gonuke** [comment]
We can watch the outcome here: https://github.com/pyne/pyne/actions/runs/1045248495

**Comment by gonuke** [comment]
Missing CHANGELOG here as well!

**Pull request merged**